src/components/Login/index.tsx

```diff
@@ -0,0 +1,64 @@
+import {
+  Modal,
+  ModalOverlay,
+  ModalContent,
+  ModalHeader,
+  ModalBody,
+  ModalCloseButton,
+  Button,
+  ModalFooter,
+} from "@chakra-ui/react";
+import { Flex, Text, Icon, Link, useDisclosure } from "@chakra-ui/react";
+import { FaGoogle, FaEnvelope } from "react-icons/fa";
+import GoogleSvg from "@/assets/svg/LoginSvg/gmail.svg";
+import EmailSvg from "@/assets/svg/LoginSvg/email.svg";
+
+function LoginModalContent(props: any) {
+  return (
+    <Modal isOpen={props.isOpen} onClose={props.onClose} isCentered size={"sm"}>
+      <ModalOverlay />
+      <ModalContent>
+        <ModalHeader>
+          <Flex flexDir="column" mt="40px" mb="20px">
+            <Text fontSize="xl" fontWeight="bold">
+              Log in or sign up
+            </Text>
+            <Text fontSize="sm">Use your email or google account to continue with CourtCanva</Text>
+          </Flex>
+        </ModalHeader>
+        <ModalCloseButton />
+        <ModalBody mb="20px">
+          <Flex flexDir="column" justifyContent="space-around" gap="25px">
+            <Button variant="loginBtn" position="relative">
+              <Icon w="28px" h="28px" position="absolute" top="11px" left="20px">
+                <FaGoogle />
+              </Icon>
+              <Text>Continue with Google </Text>
```

Comment:
Please consider if this modal should be auto-focused on the close button when it opened? 

I would say that autofocus on the first button of this modal which is "continue with Google" should be better than the close button as default. Think about if someone opened this modal and then pressed enter key, what going to happen? The modal will be closed because the autofocus is on the close button.

Focus control function please refer to this page:
https://chakra-ui.com/docs/components/overlay/modal#focus-on-specific-element

Reply:
thanks. add the property initialref